docs/pages/index.tsx

```diff
@@ -20,10 +20,8 @@ export default function Home() {
   return (
     <BrandingCssVarsProvider>
       <Head
-        title="MUI: Move faster with intuitive React UI tools"
-        description="MUI offers a comprehensive suite of free UI tools to help you ship new features faster.
-        Start with Material UI, our fully-loaded component library, or bring your own design
-        system to our production-ready components."
+        title="MUI: The React component library you always wanted"
+        description="MUI provides a simple, customizable, and accessible library of React components. Follow your own design system, or start with Material Design."
```

Comment:
A direct revert from https://github.com/mui/material-ui/commit/e7b9793afd312c338affbb58c49cd8a28bad8110#r135660732

Reply:
Lived in production with 48e7dfc278b71734df1ea94d65aa25392ca1ffbc.